TSPL.docc/ReferenceManual/Attributes.md

```diff
@@ -210,8 +210,105 @@ including important milestones.
   obsoleted: <#version number#>
   ```
   The *version number* consists of one to three positive integers, separated by periods.
+
+- The `noasync` argument indicates that the declared symbol can't be used directly
+  in an asynchronous context.
+
+  Because Swift concurrency can resume on a different thread
+  after a potential suspension point,
+  you can't use elements like thread-local storage, locks, mutexes, and semaphores
+  across suspension points. For example, accessing thread-local storage
+  on both sides of a suspension point could produce unintended behaviors.
```

Comment:
Is there something more direct we can say than just "unintended behaviors"?  Maybe something like data races or bugs or incorrect results?